daemon/daemon_unix.go

```diff
@@ -653,6 +653,9 @@ func configureKernelSecuritySupport(config *Config, driverName string) error {
 }
 
 func (daemon *Daemon) initNetworkController(config *Config, activeSandboxes map[string]interface{}) (libnetwork.NetworkController, error) {
+	if err := processPoolsConfig(config); err != nil {
+		return nil, err
+	}
```

Comment:
We're constantly running out of networks, during local development and CI. 
We're calling `docker network ls -q | xargs docker rm -fv` every 2 hours in our runners.